hypothesis-python/tests/cover/test_deadline.py

```diff
@@ -72,13 +72,13 @@ def test_deadlines_participate_in_shrinking():
     @settings(deadline=500)
     @given(st.integers())
     def slow_if_large(i):
-        if i >= 10000:
+        if i >= 100:
```

Comment:
Did this change because we no longer generated numbers which are `>= 10000`? A bit alarming if true. If not, why did it change?

Reply:
...good question.